### DISCUSSION TITLE
OpenGL application with WGL

### DISCUSSION THREAD

**Discussion by vlOd2** [itsmattkc/dotnet9x#13] [Show and tell]
I was able to get an OpenGL application running with WGL. Obviously running under .NET Framework 2.0 :P.
I plan to port Rubydung 132211 to this, so thats why it's called Rubydung 95 (because it can run under basically anything with GL lmao)
And for those curious, I am using [vmdisp9x](https://github.com/JHRobotics/vmdisp9x/), and while it is software emulation, and a slow one of that (due to 95's lack of SSE), it still runs pretty well. OpenGL demos I found ran at 100 FPS+

![image](https://github.com/itsmattkc/dotnet9x/assets/66838724/40381672-3b99-4a18-9616-10fc479a5e1f)

**Comment by vlOd2** [comment]
I think this is the most cursed any living being has done on this planet, period.
![image](https://github.com/itsmattkc/dotnet9x/assets/66838724/728a5885-037c-4372-8ac5-dfb0dd801a59)

**Reply by portalmaster137** [reply]
no this....this cant. THIS DOESN'T EXIST.

please, i need bleach
